### ISSUE TITLE
remote ssh with local server: re-login required after a few minutes

### ISSUE THREAD

**Issue by aeschli** [microsoft/vscode-remote-release#1545]
Testing https://github.com/microsoft/vscode-remote-release/issues/1521

Version: 1.39.0-insider
Commit: 313ede61cbad8f9dc748907b3384e059ddddb79a
Date: 2019-10-01T07:29:26.409Z
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Linux x64 4.15.0-58-generic

- Remote - SSH nightly
- Set `"remote.SSH.useLocalServer": true`
- Connect to an ssh host with password auth (CentOS 7)
- Opening a folder
- Opening another folder (in the same window). No log-in required (good).
- Wait a few minutes
- Log gets filled with `Server delay-shutdown request failed` entries.
- Open another folder. Log-in required.
```
[14:39:47.012] remote-ssh-nightly@2019.10.238
[14:39:47.012] linux x64
[14:39:47.013] SSH Resolver called for "ssh-remote+10.165.253.33", attempt 1
[14:39:47.013] SSH Resolver called for host: 10.165.253.33
[14:39:47.013] Setting up SSH remote "10.165.253.33"
[14:39:47.022] Acquiring local install lock: /tmp/vscode-remote-ssh-10.165.253.33-install.lock
[14:39:47.024] Looking for existing server data file at /home/aeschli/.config/Code - Insiders/User/globalStorage/ms-vscode-remote.remote-ssh-nightly/vscode-ssh-host-10.165.253.33-313ede61cbad8f9dc748907b3384e059ddddb79a/data.json
[14:39:47.024] Found local server running: {"agentPort":43043,"osReleaseId":"centos","arch":"x86_64","webUiAccessToken":"","sshAuthSock":"","pid":23258,"ipcHandlePath":"/run/user/1000/vscode-ssh-askpass-af3c4612e40507c608a72bf717ff4e3ae6c8fa46.sock","socksPort":38571,"startupTime":1569933568279}
[14:39:47.027] Found running server - short-circuiting install
[14:39:47.028] Starting forwarding server. localPort 40863 -> socksPort 38571 -> remotePort 43043
[14:39:47.028] Forwarding server listening on 40863
[14:39:47.028] Waiting for ssh tunnel to be ready
[14:39:47.029] [Forwarding server 40863] Got connection 0
[14:39:47.029] Tunneled remote port 43043 to local port 40863
[14:39:47.029] Resolved "ssh-remote+10.165.253.33" to "127.0.0.1:40863"
[14:39:47.029] TELEMETRY: {"eventName":"resolver","properties":{"outcome":"success","osReleaseId":"centos","arch":"x86_64","askedPw":"0","askedPassphrase":"0","asked2fa":"0","askedHostKey":"0","remoteInConfigFile":"0","gotUnrecognizedPrompt":"0"},"measures":{"resolveAttempts":1,"retries":1}}
[14:39:47.030] ------
[14:39:47.051] [Forwarding server 40863] Got connection 1
[14:39:47.052] [Forwarding server 40863] Got connection 2
[14:42:35.052] Server delay-shutdown request failed: connect ECONNREFUSED /run/user/1000/vscode-ssh-askpass-af3c4612e40507c608a72bf717ff4e3ae6c8fa46.sock
[14:42:37.052] Server delay-shutdown request failed: connect ECONNREFUSED /run/user/1000/vscode-ssh-askpass-af3c4612e40507c608a72bf717ff4e3ae6c8fa46.sock
[14:42:39.053] Server delay-shutdown request failed: connect ECONNREFUSED /run/user/1000/vscode-ssh-askpass-af3c4612e40507c608a72bf717ff4e3ae6c8fa46.sock
...
```

In the VM:
![image](https://user-images.githubusercontent.com/6461412/65963045-3a6e2f80-e45a-11e9-8783-9033fca7263b.png)

**Comment by roblourens** [comment]
Does that happen every time? Is it always about 3 minutes?

**Comment by aeschli** [comment]
Yes, it's around 3 minutes. I can reproduce it.

**Comment by roblourens** [comment]
If you run the "Kill local VS Code server" command and try reload the window immediately, just to make sure you are starting fresh with one window, does it still happen, and if so can you share the log from that session?

I don't know what's special about 3 minutes...

**Comment by roblourens** [comment]
Fixed in af705ffa4347f2c7511a077943865d25568dffd1
